### PR TITLE
Fixes #1155: Topic-info - Tablet (Landscape) (Highfi)

### LINKED ISSUE #1155: Topic-info - Tablet (Landscape) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/62e87a8d-1340-498f-bd0f-4fa91049def3/

Implement high-fi UI for **Topic-info** tablet landscape mode

**Target PR date**: 13 July 2020
**Target completion date**: 17 July 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1155 
- Implemented high-fi UI for **Topic-Info** for landscape mode - tablet
- Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/62e87a8d-1340-498f-bd0f-4fa91049def3/
### 1024x768 MDPI Emulator
screenshot-2020-07-10_19 38 46 602

### 2560x1800 XHDPI Emulator
screenshot-2020-07-10_19 42 58 978

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested one major change similar to `https://github.com/oppia/oppia-android/pull/1463#discussion_r453457580`

**Comment by MohamedMedhat1998** [comment]
> 
> 
> Suggested one major change similar to `https://github.com/oppia/oppia-android/pull/1463#discussion_r453457580`

Done!  
This should be merged after #1463 because the test cases are fixed there.

**Comment by rt4914** [comment]
Unassigning myself for now https://github.com/oppia/oppia-android/pull/1463#pullrequestreview-447881450

File: app/src/main/res/values-sw600dp-land/integers.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <integer name="topic_description_collapsed">100</integer>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/integers.xml]: Remove this integer. This is incorrect approach, as discussed today.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested one change.

File: app/src/main/res/values-sw600dp-land/integers.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values-sw600dp-land/integers.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**